<launch>
  <node name="tree_cluster" pkg="dbscan" type="dbscan" output="screen">
    <param name="scan_number_360" type="double" value="900.0"/>
    <param name="scan_range" type="double" value="20.0"/>
    <param name="tree_point" type="string" value="/tree_pt"/>
    <param name="scan_topic_name" type="string" value="scan"/>

    <param name="EPS" type="double" value="0.002"/>
    <param name="MinPts" type="int" value="5"/>
    <param name="tree_residual" type="double" value="0.0001"/>
    <param name="tree_radius_max" type="double" value="0.13"/>
    <param name="tree_radius_min" type="double" value="0.1"/>
  </node>
  
  <!--node name="goal_sender" pkg="dbscan" type="sendgoal2">
    <param name="scan_number_360" type="double" value="900.0"/>
    <param name="half_laser_num" type="int" value="450"/>
  </node/-->

  <!--node name="localizer" pkg="dbscan" type="localization" output="screen">
    <param name="scan_number_360" type="double" value="900.0"/>
    <param name="half_laser_num" type="double" value="450.0"/>
    <param name="laser_num" type="double" value="900.0"/>
    <param name="laser_name" type="string" value="/velodyne"/>
    <param name="base_link_name" type="string" value="/base_link"/>
    <param name="odom_name" type="string" value="/odom"/>
    <param name="map_name" type="string" value="/odom"/>
  </node/-->
  
  <!--node name="visualization" pkg="rviz" type="rviz"/-->
</launch>
